WebInkEnhancement/explainer.md

```diff
@@ -69,7 +69,9 @@ As Pointer events gets delivered to an app, application continues rendering ink,
 const renderer = new InkRenderer();
 
 try {
-    let presenter = await navigator.ink.requestPresenter('pen-stroke-tip');
+    let presenter = await navigator.ink.requestPresenter('pen-stroke-tip', canvas);
+    if (presenter.expectedImprovement < 3)

```

Comment:
Thank you for putting this together. Now that I see it, I'm still not sure what the right units would be in terms of milliseconds vs. frames, but we can keep this for now.

Let's name the constant (maybe update to 8?), and add a comment to the code. Something like:
// With pointerraw events and javascript prediction, we can reduce latency
// by 16+ms, so fallback if the InkPresenter is not capable enough to provide benefit.